static/app/components/dropdownMenu/index.spec.tsx

```diff
@@ -285,54 +266,48 @@ describe('DropdownMenu', function () {
   });
 
   it('closes after clicking external link', async function () {
+    const errorSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
+
     render(
       <DropdownMenu
         items={[{key: 'item1', label: 'Item One', externalHref: 'https://example.com'}]}
         triggerLabel="Menu"
-      />,
-      {
-        deprecatedRouterMocks: true,
-      }
+      />
     );
 
     await userEvent.click(screen.getByRole('button', {name: 'Menu'}));
     await userEvent.click(screen.getByRole('menuitemradio', {name: 'Item One'}));
     await waitFor(() => {
       expect(screen.queryByRole('menuitemradio')).not.toBeInTheDocument();
     });
+
+    // JSDOM throws an error on navigation to random urls
+    expect(errorSpy).toHaveBeenCalledTimes(1);
```

Comment:
There is already another test in this file which expects this error to be thrown, this already appeared to be an issue